src/components/button-toggle-group/__snapshots__/button-toggle-group.spec.js.snap

```diff
@@ -0,0 +1,429 @@
+            "background": "#E5EAEC",
+            "border": "#CCD6DA",
+            "button": "#e6ebed",
+            "buttonText": "rgba(0,0,0,.2)",
+            "disabled": "rgba(0,0,0,0.55)",
+            "input": "#F2F4F5",
+            "text": "rgba(0,0,0,0.3)",
+          },
+          "icon": Object {
+            "focus": "#335B6D",
+          },
+          "name": "small",
+          "shadows": Object {
+            "cards": "0 3px 3px 0 rgba(0,20,29,0.2), 0 2px 4px 0 rgba(0,20,29,0.15)",
+            "depth1": "0 5px 5px 0 rgba(0,20,29,0.2), 0 10px 10px 0 rgba(0,20,29,0.1)",
+            "depth2": "0 10px 20px 0 rgba(0,20,29,0.2), 0 20px 40px 0 rgba(0,20,29,0.1)",
+            "depth3": "0 10px 30px 0 rgba(0,20,29,0.1), 0 30px 60px 0 rgba(0,20,29,0.1)",
+            "depth4": "0 10px 40px 0 rgba(0,20,29,0.04), 0 50px 80px 0 rgba(0,20,29,0.1)",
+          },
+          "text": Object {
+            "color": "rgba(0,0,0,0.9)",
+            "placeholder": "rgba(0,0,0,0.3)",
+            "size": "14px",
+          },
+        }
+      }
+      value="foo"
+    >
+      Foo
+    </ButtonToggle>
+    <ButtonToggle
+      id="bar"
+      name="button-toggle-group"
+      size="large"
+      theme={
+        Object {
+          "colors": Object {
+            "base": "#00A376",
+            "border": "#668491",
+            "brand": "#00DC00",
+            "error": "#C7384F",
+            "focus": "#FFB500",
+            "focusedIcon": "#335B6D",
+            "focusedLinkBackground": "#FFDA7F",
+            "info": "#0073C2",
+            "primary": "#00805D",
+            "secondary": "#006045",
+            "success": "#00B000",
+            "tertiary": "#003F2E",
+            "warning": "#E96400",
+            "white": "#FFFFFF",
+            "whiteMix": "#E5F5F1",
+          },
+          "disabled": Object {
+            "background": "#E5EAEC",
+            "border": "#CCD6DA",
+            "button": "#e6ebed",
+            "buttonText": "rgba(0,0,0,.2)",
+            "disabled": "rgba(0,0,0,0.55)",
+            "input": "#F2F4F5",
+            "text": "rgba(0,0,0,0.3)",
+          },
+          "icon": Object {
+            "focus": "#335B6D",
+          },
+          "name": "small",
+          "shadows": Object {
+            "cards": "0 3px 3px 0 rgba(0,20,29,0.2), 0 2px 4px 0 rgba(0,20,29,0.15)",
+            "depth1": "0 5px 5px 0 rgba(0,20,29,0.2), 0 10px 10px 0 rgba(0,20,29,0.1)",
+            "depth2": "0 10px 20px 0 rgba(0,20,29,0.2), 0 20px 40px 0 rgba(0,20,29,0.1)",
+            "depth3": "0 10px 30px 0 rgba(0,20,29,0.1), 0 30px 60px 0 rgba(0,20,29,0.1)",
+            "depth4": "0 10px 40px 0 rgba(0,20,29,0.04), 0 50px 80px 0 rgba(0,20,29,0.1)",
+          },
+          "text": Object {
+            "color": "rgba(0,0,0,0.9)",
+            "placeholder": "rgba(0,0,0,0.3)",
+            "size": "14px",
+          },
+        }
+      }
+      value="bar"
+    >
+      Bar
+    </ButtonToggle>
+  </styled.div>
+</FormField>
+`;
```

Comment:
more than 90% of this snapshot is a theme object. Could you change the render method to `shallow` or use `TestRenderer.create` if you're checking styles?